src/Input.test.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import { render, fireEvent, screen } from '@testing-library/react';
+
+import Input from './Input';
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('Input', () => {
+  const placeholdeText = '할 일을 입력해 주세요';
+  const buttonText = '추가';
+  const handleClick = jest.fn();
+  const handleChange = jest.fn();
+
+  const renderInput = (value) => {
+    const utils = render((
+      <Input
+        value={value}
+        onClick={handleClick}
+        onChange={handleChange}
+      />
+    ));
+
+    return { ...utils };
+  };
```

Comment:
```suggestion
  const renderInput = (value) => render((
    <Input
      value={value}
      onClick={handleClick}
      onChange={handleChange}
    />
  ));
```
이렇게 하면 더 심플할 것 같네요 :)